packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageCanvas.js

```diff
@@ -34,36 +34,33 @@ function SingleImageCanvas({
           transformMatrix
         }}
       >
-        <svg
+        <g
           ref={canvasLayer}
+          data-testid='single-image-canvas-visxzoom-transform-group'
+          transform={transform}
         >
```

Comment:
I'm guessing this got broken in the move to `VisXZoom`, but you want to  use the `<g>` element as the subject canvas, since it wraps the entire subject image.

The `<svg>` element was constrained to the visible viewport, so wouldn't have given the correct dimensions, in turn leading to incorrect scale calculations for the rendered image on the screen.

Reply:
On second thoughts, I've left the parent `<svg>` in this comppnent. My original pass at fixing this removed the nested `<svg>`, but I'm not sure if it was important to establish a new SVG viewport here, so I've changed my mind and left in.

Changing the ref from the `<svg>` element to the `<g>` element should be sufficient to fix the incorrect canvas sizing in Firefox.